docs/configuration/examples.md

```diff
@@ -129,3 +129,39 @@ Customize for your identity provider run `docker-compose up -f nginx.docker-comp
 [helloworld]: https://hub.docker.com/r/tutum/hello-world
 [httpbin]: https://httpbin.org/
 [https load balancing]: https://cloud.google.com/kubernetes-engine/docs/concepts/ingress
+
+## Istio
```

Comment:
nit: do you mind putting links to the referenced products under the header (e.g. Istio, Certmanager, and Grafana?)